R/AnnotateConfidenceLevel1.R

```diff
@@ -23,30 +23,31 @@
 #' # - "MassFeature": Your unique mass feature, character.
 #' # - "mz": The mz value, numeric.
 #' # - "rt": The retention time, in seconds, numeric.
-#' # - "column": Column the mass feature was run on, character.
-#' # - "z": The ion mode, numeric.
+#' # - "column": Column the mass feature was run on, character. ##Possible options should be documented for this - HILIC vs RP? CYANO?
+#' # - "z": The ion mode, numeric. ##This is not always known - we should use polarity ("pos" vs "neg") rather than absolute charge (z)
 #' # - "MS2": MS2 data for those compounds that have it, character, in concatenated format.
 #'
-#' example_dir <- system.file("example_data", package = "phobos")
-#' example_data <- list.files(example_dir, full.names = TRUE)
-#' experimental.values <- read.csv(grep("Example_Experimental", example_data, value = TRUE))
-#' theoretical.values <- read.csv(grep("Theoretical_Data", example_data, value = TRUE))
-#' example_confidenceL1 <- AnnotateConfidenceLevel1(experimental.values = experimental.values, theoretical.values = theoretical.values,
+#' example.dir <- system.file("example_data", package = "phobos") ## This folder does not currently ship with the package, so these examples are broken
+#' experimental.values <- read.csv(dir(example.dir, full.names = TRUE, pattern = "Example_Experimental"))
+#' theoretical.values <- read.csv(dir(example.dir, full.names = TRUE, pattern = "Theoretical_Data"))
+#' example.confidence.1 <- AnnotateConfidenceLevel1(experimental.values = experimental.values, theoretical.values = theoretical.values,
 #' mz.flexibility = 0.02, rt.flexibility = 30)
 AnnotateConfidenceLevel1 <- function(experimental.values, theoretical.values, mz.flexibility, rt.flexibility) {
+  # Include tests here for correct shape and value of arguments
+  if(class(experimental.values)!="data.frame"){
+    stop("Looks like experimental.values is in the wrong format")
+  }
+  if(names(experimental.values)!=c("compound", "mz", "rt", "column", "z", "MS2")){
+    stop("Looks like experimental.values is in the wrong format")
+  }
+  # Etc.
+
+  # Variable names as a whole could use some innovation - what is the content of "My.Fuzzy.Join"?
```

Comment:
I decided to leave it as "Fuzzy.Join" but added some documentation within the function for explanation. People won't really be often digging into this anyway.